### PR TITLE
feat(duplicated_node_checker): add duplicate nodes to ignore

### PR DESCRIPTION
**Pull request by dkoldaev**
## Description
Sometimes we want to avoid publishing error diagnostics when there are multiple nodes with same name (eg. two rviz). This PR introduces a parameter with lists of nodes that should be ignored by duplicate nodes checker

## Related links

**Parent Issue:**

- Link

- [TierV internal link](https://star4.slack.com/archives/C06RT0A7VCG/p1720607575671149)

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

File: system/duplicated_node_checker/config/duplicated_node_checker.param.yaml [hunk 1]
```diff
@@ -2,3 +2,5 @@
   ros__parameters:
     update_rate: 10.0
     add_duplicated_node_names_to_msg: true # if true, duplicated node names are added to msg
+    nodes_to_ignore:  # List of nodes to ignore when checking for duplicates
+      - /rviz
```
Comment (tkimura4) [inline system/duplicated_node_checker/config/duplicated_node_checker.param.yaml:6]: In Autoware, the node name of rviz is  ```/rviz2```.
https://github.com/tier4/autoware_launch/blob/tier4/main/autoware_launch/launch/autoware.launch.xml#L129

Comment (tkimura4) [inline system/duplicated_node_checker/config/duplicated_node_checker.param.yaml:6]: Could you also create the PR for [autoware_launch](https://github.com/tier4/autoware_launch/blob/9b70c1b921d5dadbb8d20280d5348331775eb08a/autoware_launch/config/system/duplicated_node_checker/duplicated_node_checker.param.yaml)?

Comment (dkoldaev) [reply system/duplicated_node_checker/config/duplicated_node_checker.param.yaml]: Good catch, thank you! fixed here: https://github.com/autowarefoundation/autoware.universe/pull/7959/commits/945beea1f80a8c6459d71376bb09da5aecf5332d

Comment (dkoldaev) [reply system/duplicated_node_checker/config/duplicated_node_checker.param.yaml]: Thank you! I made a PR to OSS launch repo: https://github.com/autowarefoundation/autoware_launch/pull/1070

**Review by shmpwk — APPROVED** [review]
LGTM

**Pull request merged**